2021-05-06 10:22:29,278 - __main__ - INFO - Error detected
Traceback (most recent call last):
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 29, in genlog
    self.create_public()
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Cоздание_исследований\create_public.py", line 72, in create_public
    driver.find_element_by_css_selector("#itemForm\:tabView\:city_panel > ul > li:nth-child(1)").click()
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 598, in find_element_by_css_selector
    return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#itemForm\:tabView\:city_panel > ul > li:nth-child(1)"}
  (Session info: chrome=90.0.4430.93)

